.travis.yml

```diff
@@ -64,21 +64,14 @@ install:
     pip install --upgrade setuptools
   # Install only from travis wheelhouse
   - if [ -z "$PRE" ]; then
-        wheelhouse_pip_install python-dateutil $NUMPY $PANDAS pyparsing pillow sphinx!=1.3.0;
+        wheelhouse_pip_install python-dateutil $NUMPY $PANDAS pyparsing pillow sphinx!=1.3.0 mock;
```

Comment:
However, it appears all imports prefer the Python 3 location, so this should be harmless.